Package.swift

```diff
@@ -17,7 +17,7 @@ let package = Package(
     )
   ],
   dependencies: [
-    .package(url: "https://github.com/pointfreeco/swift-composable-architecture", from: "0.9.0")
+    .package(url: "https://github.com/pointfreeco/swift-composable-architecture", from: "0.21.0")
```

Comment:
is 0.21.0 the version that introduced the new style of `TestStore`? or is there some other reason to pin to this version?